assets/js/modules/analytics/components/dashboard/DashboardOverallPageMetricsWidget/index.js

```diff
@@ -0,0 +1,211 @@
+/**
+ * OverallPageMetricsWidget component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
```

Comment:
Unfortunately, although I did a `git mv` for the `DashboardOverallPageMetricsWidget*.js` -> `DashboardOverallPageMetricsWidget*/index.js` renames, GitHub hasn't recognised it.

As it was a move, I've left the datestamp in these as they were...